doc/api/console.md

```diff
@@ -7,7 +7,7 @@ JavaScript console mechanism provided by web browsers.
 
 The module exports two specific components:
 
-* A `Console` class with methods such as `console.log()`, `console.error()` and
+* A `ConsoleJefe` class with methods such as `console.log()`, `console.error()` and
```

Comment:
Yeah, it looks like this was done as a branch off the other PR branch so a lot of those other changes were carried over.